packages/bundle-size/src/utils/collectLocalReport.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs').promises;
+const glob = require('glob');
+const path = require('path');
+const { findGitRoot, findPackageRoot } = require('workspace-tools');
+
+/** @typedef {import('../utils/buildFixture').BuildResult} BuildResult */
+
+/** @typedef {BuildResult & { packageName: string }} BundleSizeReportEntry */
+/** @typedef {BundleSizeReportEntry[]} BundleSizeReport */
+
+/**
+ * @param {string} reportFile
+ *
+ * @return {Promise<[string, BuildResult[]]>}
+ */
+async function readReportForPackage(reportFile) {
+  const reportFilePath = path.resolve(process.cwd(), reportFile);
+
+  const packageName = path.basename(/** @type {string}*/ (findPackageRoot(reportFilePath)));
+  const packageReport = JSON.parse((await fs.readFile(reportFilePath)).toString());
+
+  return [packageName, packageReport];
```

Comment:
wondering why you chose to use tuple ? returning a record is most of the time the best option for both human and machine (speed of processing)

```suggestion
return { packageName, packageReport };
```

Reply:
No idea, using an object simplifies a lot, thanks for the suggestion. Applied in 2841356.